launcher/sdw_updater_gui/UpdaterApp.py

```diff
@@ -158,6 +189,26 @@ def exit_launcher(self):
         sys.exit()
 
 
+def _is_netcheck_successful() -> bool:
+    """
+    Helper function to assess network connectivity before launching updater.
+
+    Assess network connectivity by checking connection status (via nmcli) in
+    sys-net.
+    """
+    command = b"nmcli networking connectivity check"
```

Comment:
see [this comment](https://github.com/freedomofpress/securedrop-workstation/pull/743#discussion_r770889988)